### PR TITLE
add some vars for smtp backend

### PR DESCRIPTION
**Pull request by fmyzjs**
add some vars for smtp backend

### CODE REVIEW

**Comment by fmyzjs** [comment]
@feanil  please  review,thx

**Comment by feanil** [comment]
@jarv looks good to me can you please do the second review

**Comment by fmyzjs** [comment]
@feanil  It seems that @jarv  was not online

**Comment by dhpollack** [comment]
Could you add the EMAIL_FILE_PATH variable for those that want to use the filebased backend?

https://docs.djangoproject.com/en/1.6/topics/email/#file-backend

**Comment by jarv** [comment]
:+1:  needs rebase

**Comment by feanil** [comment]
@dhpollack Looks like this needs a rebase, can you please do that?

**Comment by dhpollack** [comment]
@feanil This isn't my pull request, so I assume that @fmyzjs needs to rebase it, right?

File: AUTHORS [hunk 1]
```diff
@@ -32,3 +32,5 @@ Ray Hooker <ray.hooker@gmail.com>
 David Pollack <david@sologourmand.com>
 Rodolphe Quiedeville <rodolphe@quiedeville.org>
 Matjaz Gregoric <mtyaka@gmail.com>
+Jason Zhu <fmyzjs@gmail.com>
+=======
```
Comment (feanil) [inline AUTHORS]: Why add this line?

Comment (fmyzjs) [inline AUTHORS]: I 'm sorry .I foggot to delete this line after rebase it..

**Comment by feanil** [comment]
@fmyzjs sorry for the stagnation here, looks like this needs another rebase and then I'm happy to merge it

**Comment by fmyzjs** [comment]
@feanil Please  review ,THX

**Pull request merged**